tck/src/main/java/ee/jakarta/tck/data/standalone/entity/EntityTests.java

```diff
@@ -966,17 +967,17 @@ public void testCursoredPageOf7FromCursor() {
 
         CursoredPage<NaturalNumber> nextPage;
         try {
-            nextPage = positives.findByFloorOfSquareRootNotAndIdLessThanOrderByBitsRequiredDesc(6L, 50L,
+            nextPage = positives.findByFloorOfSquareRootNotAndIdLessThanOrderByNumBitsRequiredDesc(6L, 50L,
                                                                                                 page.nextPageRequest());
-        } catch (MappingException x) {
-            // Test passes: Jakarta Data providers must raise MappingException when the database
+        } catch (UnsupportedOperationException x) {
+            // Test passes: Jakarta Data providers must raise UnsupportedOperationException when the database
             // is not capable of cursor-based pagination.
             return;
         }
 
         assertEquals(Arrays.toString(new Long[] { 10L, 9L, // 4 bits required, square root rounds down to 3
                                                   7L, 6L, 5L, 4L, // 3 bits required, square root rounds down to 2
-                                                  3L, 2L // 2 bits required, square root rounds down to 1
+                                                  3L // 2 bits required, square root rounds down to 1
```

Comment:
This test case asks for pages of size 7, but mistakenly tries to assert that this page will have 8 elements.